docs/howto/set-config.rst

```diff
@@ -73,6 +91,7 @@ The universal order of precedence for all configuration options is as follows:
 #. set as a command environment variable
```

Comment:
```suggestion
#. set as a command environment variable
#. set as a secret environment variable
```
Though the wording on this isn't that clear :(

Reply:
fixed in  032d33f